src/CMakeLists.txt

```diff
@@ -46,8 +46,8 @@ add_library(progress
   prg_subgraphloop_mod.F90
   prg_system_mod.F90
   prg_timer_mod.F90
-  prg_xlbo_mod.F90
-  prg_xlkernel_mod.F90)
+  prg_xlbo_mod.F90)
+  #prg_xlkernel_mod.F90)
```

Comment:
Are those function in `prg_xlkernel_mod` useful? Maybe they just need some updating as I have tried (still incomplete) in lanl/qmd-progress#194?

Reply:
They might be at some point. We will look into them with @AndersNiklasson